mbofui/app/views/popup.html

```diff
@@ -1,18 +1,28 @@
 <div>
   <div>
     <strong>
-      {{event.category}}
+      Event title would go here
```

Comment:
Do we have an event title in the db? See:
https://itsjira.umms.med.umich.edu/browse/TLMBLGIS-71

Till we do this is a placeholder.